canonical-websites/www.ubuntu.com
# 2662: Move /about/about-ubuntu/licensing to /licensing

## Done

- Moved /about/about-ubuntu/licensing to /licensing
- Updated copy according to [copydoc](https://docs.google.com/document/d/1cqzDlgJERp7BuNO-yWKz96hAQp8W4-8_5ETb_Il6x4I/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/licensing
- Check that it matches the [copydoc](https://docs.google.com/document/d/1cqzDlgJERp7BuNO-yWKz96hAQp8W4-8_5ETb_Il6x4I/edit)
- Check that /about/about-ubuntu/licensing redirects to /licensing

## Issue / Card

Fixes #2662
